Api/namedPipe/namedPipe.h

```diff
@@ -0,0 +1,17 @@
+// From http://stackoverflow.com/questions/2784500/how-to-send-a-simple-string-between-two-programs-using-pipes
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <sys/time.h>
+#include <unistd.h>
+
+
+#define BLOCK 1024 
+
+int openNamedPipe(char * something);
+
+void writeNamedPipe(int fd, void * data, int size);
```

Comment:
Didn't he tell us that `fd` shouln't be shown here. Perhaps we could change its name